lblod/frontend-gelinkt-notuleren
# Add 'edit rdfa' mode, hard-coded to be switched off

### Overview
Wire in the UI elements and fix some bits along the way.

There is a problem with some of the `besluit` nodes, as these need to be migrated to be rdfa aware. These will be in the plugins repo though.

##### connected issues and PRs:
Requires https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/379
In turn, requires https://github.com/lblod/ember-rdfa-editor/pull/1128

### Setup
Shouldn't need any as package.json has the git commits. It will take an age to `npm i` though. I think we should just dump this idea to be honest and just link...

### How to test/reproduce
All behaviour *should* be the same. There are some problems with `besluit` nodes though, which also has the side-effect of asking you each time, what decision type you want to use.

Then edit `{regulatory-statements,agendapoints}/edit.hbs` to switch `@shouldEditRdfa` to `true`. You should now see the new UI elements in the sidebar and be able to use them correctly.

### Challenges/uncertainties
Need to do more work in the plugins repo.

### Checks PR readiness
